source/MultiGrid/MultiGrid.js

```diff
@@ -433,7 +434,7 @@ export default class MultiGrid extends Component {
         columnCount={columnCount - fixedColumnCount}
         columnWidth={this._columnWidthRightGrid}
         height={this._getBottomGridHeight(props)}
-        onScroll={this._onScroll}
+        onScroll={(info) => {this._onScroll(info);onScroll && onScroll(info)}}
```

Comment:
How about moving this into the `_onScroll` method? That way the code's a bit easier to read _and_ we avoid creating a new wrapper function on each render. :)